binaries/conda/build_packages.py

```diff
@@ -15,6 +15,13 @@
     else f"$HOME/miniconda/condabin/conda"
 )
 
+CONDA_PACKAGES_PATH = os.path.join(REPO_ROOT, "binaries", "conda", "output")
+CONDA_LINUX_PACKAGES_PATH = os.path.join(
+    REPO_ROOT, "binaries", "conda", "output", "linux-64"
+)
+PACKAGES = ["torchserve", "model-archiver", "workflow-archiver"]
+PLATFORMS = ["linux-64", "osx-64", "win-64"]
```

Comment:
Add a comment that new platforms can be supported here or maybe even link the fully supported list from the convert command

Reply:
Done